docs/manual/advanced-features.tex

```diff
@@ -1140,7 +1140,8 @@
 \item \refqualclass{checker/lock/qual}{Holding}
 \end{itemize}
 
-The set of permitted expressions is a subset of all Java expressions:
+The set of permitted expressions is a subset of all Java expressions,
+with a few extensions, notably \code{<self>} and formal parameters like \<\#1>.
```

Comment:
Same concern about `<self>` as above.

Reply:
It is important to include `<self>` here, because this section is about the Checker Framework Java expression syntax.

Reply:
But it is only the Lock Checker that supports the `<self>` expression: https://github.com/typetools/checker-framework/blob/master/checker/src/main/java/org/checkerframework/checker/lock/LockVisitor.java#L78
So it seems rather confusing to include it like it is a generally supported expression.

Reply:
I added a qualifier to its description.